src/lib/resources.js

```diff
@@ -124,4 +126,8 @@ function isCoverage(organized) {
   return(organized.countOfType("Coverage") > 0);
 }
 
+function isImmunizationHistory(organized) {
+  return(organized.countOfType("Immunization") > 0);
```

Comment:
This is a-ok for now given that it's checked after the Patient Summary bit, but I may come back and revisit this logic later. I plan to put in a fallback renderer that just tries to render any set of resources and so might try to limit this to ones that are all Immunizations + a Patient or something like that ... just a note for later.

Reply:
Oh also one more ask --- can you rebase this edit against the "develop" branch? Patrick asked that we keep a develop branch that we can prop to the test site and then just use the main branch for production.

Reply:
Sure thing! I expanded the logic of `isImmunizationHistory` to check for one patient and >0 histories. I also rebased the PR onto the `develop` branch.